app/components/meetings/api.js

```diff
@@ -272,6 +272,45 @@ export default class Api
     await axios.put(url, file, config);
   }
 
+  async getMeetingDocuments(options) {
+    const documents = await this.http.get(`/api/v2016/documents`, options).then(res => res.data).catch(tryCastToApiError);
+    return documents;
+  } 
+
+  async getMeetings(options) {
+    const meetings = await this.http.get(`/api/v2016/meetings`, options).then(res => res.data).catch(tryCastToApiError);
+    return meetings;
+  }
+
+  async getNotifications(options) {
+
+    const queryParameters = {
+      
+      params : {
+          q : `symbol_s (${codes.map(solr.escape).join(' or ')})`,
+          fl : "id, symbol_s,reference_s,title_t,date_dt,url_ss",
+      }
+    }
+    const notifications = await api.queryNotifications(queryParameters).then(res => res.data).catch(tryCastToApiError);
+    return notifications;
+  }
+
+  async queryNotifications({q: userQ, ...otherParams}) {
+    let q = 'schema_s:notification'
+
+    if(userQ) q = `${q} AND (${userQ})`;
+
+    const params = { ...otherParams, q };
+
+    const notifications = await this.http.get('/api/v2013/index', { params }).then(res => res.data).catch(tryCastToApiError);
+    return notifications;
+  }
+
+  async getDecisions(options) {
+    const decisions = await this.http.get('/api/v2021/decisions', options).then(res => res.data).catch(tryCastToApiError);
+    return decisions;
+  }
```

Comment:
same as for queryMeetingDocuments. Don't pass option to be apply directly to the http library but pass parameters to filter your quer. you can have controlled option (eg cache)
```suggestion
  async queryDecisions({ cache, ...params}) {
  cache = !!cache;//force bool
    const decisions = await this.http.get('/api/v2021/decisions', { params, cache }).then(res => res.data).catch(tryCastToApiError);
    return decisions;
  }
```